packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -146,9 +146,22 @@ class WebAssetServer implements AssetReader {
   final Map<String, String> _modules;
   final Map<String, String> _digests;
 
+  // The generation number that maps to the number of hot restarts. This is used
+  // to suffix a query to source paths in order to cache-bust.
+  int _hotRestartGeneration = 0;
+
   int get selectedPort => _httpServer.port;
 
-  void performRestart(List<String> modules) {
+  /// Given a list of [modules] that need to be loaded, compute module names and
+  /// digests.
+  ///
+  /// If [writeRestartScripts] is true, writes a list of sources mapped to their
+  /// ids to the file system that can then be consumed by the hot restart
+  /// callback.
+  // TODO(srujzs): We should probably include source maps here to reload as well
```

Comment:
Should we have an issue associated with this TODO?

Reply:
This is actually a stale TODO, so I'm removing it. I didn't realize we're already storing the new source maps in the filesystem on a recompile. The browser fetches the map based on the filename, so there's nothing for us to do here.